pkg/agent/types/multicluster.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"net"
+
+	binding "antrea.io/antrea/pkg/ovs/openflow"
+)
+
+// ServiceGroupInfo is used in AntreaProxy for Antrea Multi-cluster.
+// It stores local exported Service's GroupID and its ClusterIP.
+type ServiceGroupInfo struct {
```

Comment:
Why it causes cyclic import if we put `ServiceGroupInfo` in `pkg/agent/openflow`? I saw the type is used only in `pkg/agent/openflow` and `pkg/agent/proxy", and the latter already imports the former.

Reply:
The reason is the mock file in `pkg/agent/openflow` will also need `ServiceGroupInfo`, if we place it in `pkg/agent/openflow`, then the sub package will import a parent package.

Reply:
In this case, maybe we should add a "types" package under `pkg/agent/openflow`.

Reply:
Done, I will move all `agent/openflow` types into the new folder after this PR is merged considering there will be lots of import path change.

Reply:
Sure. No need to move all types. Just types and interfaces consumed by other packages.